src/coreclr/jit/compiler.cpp

```diff
@@ -390,7 +390,7 @@ var_types Compiler::getJitGCType(BYTE gcType)
 //    true if the given struct type contains only one pointer-sized integer value type,
 //    false otherwise.
 //
-bool Compiler::isTrivialPointerSizedStruct(CORINFO_CLASS_HANDLE clsHnd) const
+bool Compiler::isTrivialPointerSizedStruct(CORINFO_CLASS_HANDLE clsHnd)
```

Comment:
I know it's a draft, but what caused you to remove the `const`?

Reply:
It's a temporary change needed for the `FieldInfo` commit - that will be reverted once SPMI data has been collected.

The nominal reason is that `RecordStructFieldInfo` is not `const` (and cannot be).